timescale-forge/vpc-peering.md

```diff
@@ -0,0 +1,269 @@
+propagate.
+
+As mentioned on the confirmation modal, you will need to update your connection string
+in order to connect to your service after migration. The `Service URL` back on the
+service details page is already updated to include the new DNS info, and should be used
+for connecting to your service.
+
+<highlight type="tip">
+When migrating your service into a VPC, ensure that your AWS VPC's security groups
+allow network access from your AWS VPC to the Forge VPC which your service has been
+migrated into. Security group configuration was previously covered as part of
+peering connection setup. Double-check to be sure, otherwise you will not be able
+to connect to your Timescale Forge service.
+</highlight>
+
+### Migrate between VPCs
+When your service is already attached to a VPC, you have the option to migrate that
+service to another VPC within the same project.
+
+<img class="main-content__illustration" src="migrate-between-vpcs.png" alt="Migrate between VPCs"/>
+
+To migrate between VPCs, expand the `Migrate into another VPC` menu and select the VPC
+to migrate your service to. Then click `Migrate`. You will then be prompted to confirm
+the migration.
+
+<img class="main-content__illustration" src="migrate-between-vpcs-confirm.png" alt="Migrate between VPCs confirmation"/>
+
+Your service will now go through a series of asynchronous operations ultimately
+resulting in your service being detached from its previous VPC and attached to the
+new VPC which you selected.
+
+In the case of VPC to VPC migration, the `Service URL` connection string will not
+be updated, only the IP address which the DNS name is associated with will be updated.
+Though the connection string does not need to be updated, please provide a few minutes
+for the DNS record changes to propagate.
+
+<highlight type="tip">
+When migrating your service between VPCs, ensure that your AWS VPC's security groups
+allow network access from your AWS VPC to the Forge VPC which your service has been
+migrated into. Security group configuration was previously covered as part of
+peering connection setup. Double-check to be sure, otherwise you will not be able
+to connect to your Timescale Forge service.
+</highlight>
+
+### Migrate back to public network
+When your service is already attached to a VPC, you have the option to migrate that
+service back to the public network.
+
+<img class="main-content__illustration" src="migrate-back-to-public.png" alt="Migrate back to public network"/>
+
+To migrate your service back to the public network, click `Migrate back to public network`.
+You will then be prompted to confirm the migration.
+
+<img class="main-content__illustration" src="migrate-back-to-public-confirm.png" alt="Migrate back to public network confirm"/>
+
+Your service will now go through a series of asynchronous operations ultimately
+resulting in your service being detached from its previous VPC and made accessible
```

Comment:
Might be okay for `being detached` here since it avoids us having to do `being made accessible`

Reply:
Changed it completely `:)`. Thanks for all of the great feedback @ncalibey!